### DISCUSSION TITLE
Getting db errors during migration from 1.24 to 1.25

### DISCUSSION THREAD

**Discussion by ericlau-solid** [dani-garcia/vaultwarden#2557] [Q&A]
Getting the following errors:

![image](https://user-images.githubusercontent.com/8011182/172973856-ee10eba3-87dd-4135-9a8f-623b7832d090.png)

I am completely stuck. The performance issues I had in 1.24 are back, so I tried to upgrade, and now upgrade is stuck, so I have no way of getting bitwarden server back up. Anything I can do? Devices new user seems interesting, but not sure what I can do.

**Comment by BlackDex** [comment]
I suggest to check this comment https://github.com/dani-garcia/vaultwarden/issues/2504#issuecomment-1140281453

In short, run the 1.24.0 version, login into the admin interface, Deauthorize sessions of all users, then upgrade to 1.25.0.

But, first backup the database, just in case.

**Comment by aquanow-eric** [comment]
I see. I didn't try that. Instead I exported to unencrypted JSON all the vaults out of 1.24, install clean 1.25, and import them. I find that JSON encrypted didn't work even though I didn't change the pwd.

At any event this did the trick. I also resolved write performance issues I had earlier as well. So all good.

**Comment by aquanow-eric** [comment]
We can consider this issue closed.
